WSL/file-permissions.md

```diff
@@ -54,11 +53,11 @@ If the file has metadata present, we simply use those metadata values instead of
 
 The result depends on if the file already has existing metadata.
 
-##### **The file does not have metadata (default)**
+##### chmod file does not have metadata (default)
 
 Chmod will only have one effect, if you remove all the write attributes of a file then the 'read only' attribute on the Windows file will be set, since this is the same behaviour as CIFS (Common Internet File System) which is the SMB (Server Message Block) client in Linux.
 
-##### The file has metadata
+##### chmod file has metadata

```

Comment:
Same comment as above
```suggestion
##### The file has metadata

```

Reply:
We can't reuse the same header over and over like this